### PR TITLE
update web pages for tabbed examples

### PR DESCRIPTION
**Pull request by sav-norem**
to get tabbed examples in CLI and client languages, we need to update each data type page to include hugo shortcodes and updated CLI examples using bicycles for a cohesive examples experience

### CODE REVIEW

File: docs/docs/bloom-filter.md [hunk 1]
```diff
@@ -71,57 +71,33 @@ Using Redis Stack's Bloom filter for this type of application provides these ben
...
 
-```
-> BF.RESERVE customFilter 0.0001 600000
-OK
-```
-
-```
-> BF.MADD customFilter foo bar baz
-```
+There is always a chance that even with this few items, there's a collision and we get a false positive.
```
Comment (dwdougherty) [inline docs/docs/bloom-filter.md:98]:
```suggestion
Note: there is always a chance that even with just a few items, there could be a collision, yielding a false positive.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
A few suggested changes.

**Comment by sav-norem** [comment]
@dwdougherty updated bloom and accidentally put the t-digest update on this branch, so [the t digest PR](https://github.com/RedisBloom/RedisBloom/pull/687) would also be resolved with this PR

File: docs/docs/bloom-filter.md [hunk 2]
```diff
@@ -71,57 +71,30 @@ Using Redis Stack's Bloom filter for this type of application provides these ben
...
-OK
-```
-
-```
-> BF.MADD customFilter foo bar baz
-```
+Note: there is always a chance that even with just a few items, there could be a collision, yielding a false positive.
 
-## Sizing Bloom filters
+## Reserving Bloom filters
```
Comment (LiorKogan) [inline docs/docs/bloom-filter.md:97]: Bloom is always with a capital 'B' (named after Burton Howard Bloom).

File: docs/docs/t-digest.md [hunk 3]
```diff
@@ -62,62 +62,56 @@ You measure the IP packets transferred over your network each second and try to
...
 
 #### Estimating fractions or ranks by values
 
 Another helpful feature in t-digest is CDF (definition of rank) which gives us the fraction of observations smaller or equal to a certain value. This command is very useful to answer questions like "*What's the percentage of observations with a value lower or equal to X*".
 
->More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X plus half the number of observations that are equal to X
-
-Let's illustrate this with an example: if we have a set of observations of people's age with gaussian distribution, we can ask a question like "What's the percentage of people younger than 50 years?"
+>More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X plus half the number of observations that are equal to X. We can also use the `TDIGEST.RANK` command, which is very similar. Instead of returning a fraction, it returns the ----estimated---- rank of a value. The `TDIGEST.RANK` command is also variadic, meaning you can send one number to query or multiple.
```
Comment (LiorKogan) [inline docs/docs/t-digest.md:83]: Consider changing
> you can send one number to query or multiple
to:
you can use a single command to retrieve estimations for one or more values

File: docs/docs/t-digest.md [hunk 4]
```diff
@@ -62,62 +62,56 @@ You measure the IP packets transferred over your network each second and try to
...
 
-```
-> TDIGEST.ADD my-tdigest 45.88 44.2 58.03 19.76 39.84 69.28 50.97 25.41 19.27 85.71 42.63
+Here's an example. If you have a set of observations of people's ages, you can ask a question like "What's the percentage of bike racers that are younger than 50 years?"
```
Comment (LiorKogan) [inline docs/docs/t-digest.md:85]: Consider changing "if you have a set of observations of people's ages" to "Given a set of biker's ages"

File: docs/docs/bloom-filter.md [hunk 5]
```diff
@@ -71,57 +71,30 @@ Using Redis Stack's Bloom filter for this type of application provides these ben
...
 
-```
-> BF.RESERVE customFilter 0.0001 600000
-OK
-```
-
-```
-> BF.MADD customFilter foo bar baz
-```
+Note: there is always a chance that even with just a few items, there could be a collision, yielding a false positive.
```
Comment (LiorKogan) [inline docs/docs/bloom-filter.md:95]: I'm not sure if 'collision' is clear here. You need to understand how a Bloom filter works to understand what collision means, and probably we don't want to explain it here. I would change it to "there could be a false positive" (and maybe this term should be explained here as well).

Comment (sav-norem) [reply docs/docs/bloom-filter.md:96]: @dwdougherty as well - how does "Note: there is always a chance that even with just a few items, there could be a false positive, meaning an item could "exist" even though it has not been explicitly added to the Bloom filter. For a more in depth understanding of the probabilistic nature of a Bloom filter, check out the blog posts linked at the bottom of this page."

Comment (dwdougherty) [reply docs/docs/bloom-filter.md:96]: LGTM

**Comment by sav-norem** [comment]
@LiorKogan I removed the top k leaderboard use case

**Review by LiorKogan — APPROVED** [review]

**Pull request merged**